projects/wave-core-new/src/lib/datatable/table/full-display/full-display.component.html

```diff
@@ -0,0 +1,17 @@
+<wave-dialog-header>Full coordinates</wave-dialog-header>
+<mat-dialog-content>
+    <table>
+        <thead>
+            <th>X</th>
+            <th>Y</th>
+        </thead>
+    <tr *ngFor="let x of xCoords, index as i">
+        <td>{{x}}</td>
+        <td>{{yCoords[i]}}</td>
+    </tr>   
+    </table>
```

Comment:
We could have lots of coordinates, so we should use either a virtual scroll (https://material.angular.io/cdk/scrolling/overview) or a table with pagination (https://material.angular.io/components/table/examples).